javascript/javascript1/week2/class07Students.js

```diff
@@ -0,0 +1,52 @@
+// copy your homework solution here to make the webpage work!
+
+const class07Students = [];
+let count = 0;
+function addStudentToClass(studentName) {
+   if (studentName.toLowerCase() == 'queen') {
+class07Students.push(studentName);
+  }
+  else if ((class07Students.indexOf(studentName) !== -1) || (studentName == '' ) ){
```

Comment:
```js
  else if(studentName === '') {
    return 'Name cannot be empty';
  }
  else if (class07Students.includes(studentName)){
```

Reply:
There was a message missing for empty string. 
And you can use the `includes` array method to make this simpler.